examples/platforms/cc2538/uart.c

```diff
@@ -54,14 +54,22 @@ static void processTransmit(void);
 static const uint8_t *sTransmitBuffer = NULL;
 static uint16_t sTransmitLength = 0;
 
+typedef struct RecvBuffer
+{
+    uint16_t Offset;
+    uint16_t Length;
+} RecvBuffer;
+
 static uint8_t sReceiveBuffer[kReceiveBufferSize];
```

Comment:
We don't have a good way to estimate worst-case time.  It would be useful to maintain a counter of bytes dropped due to insufficient buffer space and have a way to retrieve that count over spinel.